tests/test_keydb.py

```diff
@@ -43,7 +43,7 @@
 
 # Generate the three keys to use in our test cases.
 KEYS = []
-for junk in range(3):
+for junk in range(4):
```

Comment:
```suggestion
for junk in range(3):
```
I think we don't need this change now that we've dropped the WIP test in 376590d4e82ecf93e40157a7178ed27e5bb254fe?